.code-samples.meilisearch.yaml

```diff
@@ -1158,3 +1158,17 @@ update_proximity_precision_settings_1: |-
 reset_proximity_precision_settings_1: |-
   curl \
     -X DELETE 'http://localhost:7700/indexes/books/settings/proximity-precision'
+index_settings_tutorial_api_get_setting_1: |-
+  curl \
+    -X GET 'http://localhost:7700/indexes/INDEX_NAME/settings/searchable-attributes'
+index_settings_tutorial_api_put_setting_1: |-
+  curl \
+    -X PUT 'http://localhost:7700/indexes/INDEX_NAME/settings/searchable-attributes' \
+    -H 'Content-Type: application/json' \
+    --data-binary '[
+      "title",
+      "overview"
+    ]'
+index_settings_tutorial_api_task_1: |-
+curl \
+  -X GET 'http://localhost:7700/tasks/TASK_UID'
```

Comment:
`TASK_UID` will not be a valid task uid (contrary to `INDEX_NAME` for the index uid), should we keep it as it is?

Reply:
I would prefer to keep it as is. I prefer a user sees an explicit error because they copypasted without replacing the placeholder, than the example including an arbitrary uid that might correspond to an unrelated task (e.g. the code sample uses uid `1` and `1` corresponds to some settings they updated before starting the tutorial).